ERROR_MESSAGES.md

```diff
@@ -43,8 +43,7 @@ Any text in the following format `(Example)` are considered variables to be fill
 * 36 : CT - Must contain one Measure (eCQM) section
 * 37 : CT - Must contain correct number of performance rate(s). Correct Number is `(Expected value)`
 * 38 : CT - This `(Numerator or Denominator)` Node does not have any child Nodes
-* 39 : CT - This `(Numerator or Denominator)` Node does not have an Aggregate Count Node
-* 40 : CT - This `(Numerator or Denominator)` Node has too many child Nodes
+* 39 : CT - This `(Numerator or Denominator)` Node must have exactly one Aggregate Count node
```

Comment:
And now... there IS no ErrorCode 40. Cue the spooky campfire story.

Reply:
Are you afraid of the dark?